New package: TensorTimeSteps v0.1.3

JuliaRegistrator opened the pull request:
- Registering package: TensorTimeSteps
- Repository: https://github.com/BenjaminDecker/TensorTimeSteps.jl
- Created by: @BenjaminDecker
- Version: v0.1.3
- Commit: 99eba359457ff9ea039b11bc3c51a1092e43a2a9
- Git reference: HEAD

BenjaminDecker commented:
Thanks for your comment! 
I hope the README I added is sufficient. I plan to look into a Documenter workflow when I add more functions to the package.

If everything is fine for now, can I trigger an update to this via Registrator?

JuliaHubRegistrator committed:
New package: TensorTimeSteps v0.1.3

UUID: 1f4dac17-e489-46a5-8247-1c82af6382b6
Repo: https://github.com/BenjaminDecker/TensorTimeSteps.jl.git
Tree: 70e3421d34a07795a79f8547b17e0a601d90d3dc

Registrator tree SHA: be588f25c435fa5b456b986adf2f5c02bd3298f2
